doc/QuickStart.md

```diff
@@ -54,14 +82,24 @@ connection.
 Here we clone the data-directory for two PBot instances, naming them after the
 IRC network they will connect to:
 
-    cp -r data freenode
-    cp -r data ircnet
+    $ su pbot
+    $ cd /home/pbot
+    $ git clone https://github.com/pragma-/pbot.git
```

Comment:
Users are already expected to know how to use git on their operating system. This QuickStart guide is strictly for PBot-specific configuration.